scripts/check_compliance.py

```diff
@@ -737,17 +739,18 @@ def run(self):
             .splitlines()
 
         # Filter out everything but Python files
-        py_files = filter_py(files)
+        py_files = filter_py(GIT_TOP, files)
```

Comment:
Could do this maybe, so that `filter_py()` can keep taking just a list of paths (otherwise, the doc comment for it should be updated).

```python
py_files = filter_py(os.path.join(GIT_TOP, path) for path in files)
```

Reply:
Maybe the `git diff` could be run in `GIT_TOP` too, even though it doesn't matter for the output (since the current directory is assumed to be inside the repository you want to check). Might be less confusing.

Reply:
I've considered running `git diff` in `GIT_TOP`, however it gives the false impression that it makes a difference - it doesn't. It does only when using `git diff --relative`.

Reply:
Alright, fine with me.

Reply:
> (otherwise, the doc comment for it should be updated).

I went for updating the filter_py() comment(s) to stay farther from any command line limit.

Reply:
Alright. Will get the same list from `filter_py()` in both cases though.